src/EnergyPlus/FanCoilUnits.cc

```diff
@@ -2244,51 +2248,72 @@ namespace FanCoilUnits {
 				ControlOffset = FanCoil( FanCoilNum ).ColdControlOffset;
 
 				// get the maximum output of the fcu
-				Calc4PipeFanCoil( FanCoilNum, ControlledZoneNum, FirstHVACIteration, QUnitOutMax );
-
-				if ( QUnitOutMax < 0.0 ) { // protect against QUnitOutMax = 0 (no water flow, plant off, etc.)
-					// calculate the PLR, if load greater than output, PLR = 1 (output = max)
-					PLR = std::abs( QZnReq / QUnitOutMax );
-					if ( PLR > 1.0 ) PLR = 1.0;
-
-					// adjust the PLR to meet the cooling load calling Calc4PipeFanCoil repeatedly with the PLR adjusted
-					while ( std::abs( Error ) > ControlOffset && std::abs( AbsError ) > SmallLoad && Iter < MaxIterCycl && PLR != 1.0 ) {
-						// the water flow rate is at the maximum flow rate times the PLR
-						//    Node(FanCoil(FanCoilNum)%ColdControlNode)%MassFlowRate = PLR * FanCoil(FanCoilNum)%MaxColdWaterFlow
-						mdot = PLR * FanCoil( FanCoilNum ).MaxColdWaterFlow;
```

Comment:
The old iteration method adjusted water flow rate proportional to PLR. However, if fan speed is above 1, shouldn't water flow rate remain at the max? The new routine does the same thing but it looks wrong. For example, whatever load can be met at fan speed = 1 and full water flow is a limit for first stage fan. If the load is slightly higher than that then water flow should remain constant and air flow should be adjusted to meet the load. I'll wait and see what CI returns before I do anything here.